starknet-rpc-test/Cargo.toml

```diff
@@ -8,9 +8,8 @@ edition = "2021"
 
 anyhow = "1.0.72"
 assert_matches = "1.5.0"
-derive_more = "0.99.17"
+async-lock = { git = "https://github.com/smol-rs/async-lock", branch = "master" }
```

Comment:
Use a specific release/commit instead of a branch/tag. Usually those are deleted

Reply:
done